raiden/tests/utils/mediation_fees.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from raiden.tests.utils import factories
+from raiden.utils.mediation_fees import prepare_mediation_fee_config
+from raiden.utils.typing import ProportionalFeeAmount
+
+
+@pytest.mark.parametrize("flat_fees", [(42, 21), (43, 21)])
+def test_prepare_mediation_fee_config_flat_fee(flat_fees):
+    cli_flat_fee, expected_channel_flat_fee = flat_fees
+    token_network_address = factories.make_token_network_address()
+    fee_config = prepare_mediation_fee_config(
+        cli_token_network_to_flat_fee=((token_network_address, cli_flat_fee),),
+        proportional_fee=ProportionalFeeAmount(0),
+        proportional_imbalance_fee=ProportionalFeeAmount(0),
+    )
+
+    assert fee_config.get_flat_fee(token_network_address) == expected_channel_flat_fee
+
+
+@pytest.mark.parametrize(
+    "prop_fees",
+    [
+        (1_000_000, 1_000_000),  # 100%
```

Comment:
@karlb This doesn't make any sense. I think we should cap the fees at a way lower value.

Reply:
If I really want to send a single wei, I could be willing to pay many times the payment amount in fees. Choosing a limit is pretty arbitrary. I would rather limit things on the initiator or PFS side than in the mediator. But I don't have a strong opinion on this, so feel free to propose a different limit.